.gitignore

```diff
@@ -1 +1,2 @@
 /doc/tags
+.*.swp
```

Comment:
Friendly tip: some maintainers don't like editor-specific entries in their `.gitignore` files.  Also, this won't capture secondary, tertiary, and, uh... quadriary+ (or something) swap files.

Anyways, my tip is: in order to ignore vim swap files across all your git repos create a file `~/.config/git/ignore` with the content `.*.sw*`.